### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by cicscareers** [cicscareers/320-S20-Track1#386]
When tags are added to the filters, creating an appointment fails. This is likely due to either a naming issue between the front end and backend, or something being formatted wrong

**Comment by cicscareers** [comment]
<img width="890" alt="Appt Tagging Issue" src="https://user-images.githubusercontent.com/33036583/85869801-33324300-b79a-11ea-8426-508af296a4cc.png">

**Comment by cicscareers** [comment]
Tried to retest and was not able to recreate issue.

**Comment by powellsl99** [comment]
Yeah it is in the Trello as high priority. I believe this is a naming issue or type issue with one of the parameters for the endpoint. @victoriacaruso could you send me what they are?

**Comment by victoriacaruso** [comment]
student_id, time_of_appt, duration, selected_tags, medium, location, specialization, comment, override, student_email, supporter_id, supporter_email

**Comment by victoriacaruso** [comment]
was this adding supporter specialties to one specific appointment? 
also when this occured did it give some kind of error message like "Update to specializations_for_appointment failed:" also do you remember if it was for one specialization added or multiple?
